api/v1alpha1/containerimagebuild_types.go

```diff
@@ -120,31 +121,42 @@ type ContainerImageBuildSpec struct {
 	PluginData map[string]string `json:"pluginData"`
 }
 
+// ContainerImageBuildState defines a previously observed state of ContainerImageBuild
+type ContainerImageBuildState struct {
+	State           BuildState `json:"state,omitempty"`
+	DurationSeconds string     `json:"durationSeconds,omitempty"`
```

Comment:
Does `DurationSeconds` make more sense the current time?

Reply:
Hmm good question. How about both?

Reply:
Actually, yeah, did this on purpose since the info is actually encoded in the object mostly anyway (startedAt for building & completedAt for completed / failing, only thing missing is `Initializing`). Probably a question of what this should track? State transitions or time spent in a state. I'm going to pull it out of this PR for now.